ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -199,8 +199,8 @@ StatusDialog {
                 holdingSelector.selectedItem = entry
             } else {
                 const entry = SQUtils.ModelUtils.getByKey(
-                                popup.store.collectiblesModel,
-                                "uid", popup.preSelectedHoldingID)
+                                popup.collectiblesStore.allCollectiblesModel,
+                                "symbol", popup.preSelectedHoldingID)
```

Comment:
do we need to use `symbol`? As a target we should use `key` as unique identifier consistently everywhere.

Reply:
to me it looks correctly, in nim this is `uid` and in qml it was remapped tp `symbol` in AppLayouts/Wallet/stores/CollectiblesStore.qml

Reply:
@Seitseman is right, the `uid` is renamed in the store. 
```
    readonly property var allCollectiblesModel: RolesRenamingModel {
        objectName: "allCollectiblesModel"

        sourceModel: root._allCollectiblesModel

        mapping: [
            RoleRename {
                from: "uid"
                to: "symbol"
            }
        ]
    }
```

Reply:
The key point here is that's wrong even if it works now :upside_down_face: We should aim the state where all models have unique `key` string role and ui uses it only for identification.

Reply:
It's not necessary to be all fixed now, just to keep in mind and be consistent about that.